covid19_sfbayarea/news/base.py

```diff
@@ -55,7 +62,12 @@ def load_html(self, url: str) -> str:
     def parse_page(self, html: str, url: str) -> List[NewsItem]:
         raise NotImplementedError()
 
+    def _in_time_range(self, candidate: NewsItem) -> bool:
+        time = candidate.date_published
+        return time < self.to_date and (not self.from_date or
```

Comment:
Should the first comparison be
`time <= self.to_date` ?

Reply:
Oh, this was intentional, but I’m happy to change it.

Reply:
(Worth noting that everything here is actually a `datetime`, so if `to_date` was specified, we’re not saying “before today,” we’re saying “before right now.”)

Reply:
I didn't catch that, sorry! I thought it would return `before today`. Well, we really will have `only the latest news`. Thanks for changing it.